website/pages/docs/integrations/vault-integration.mdx

```diff
@@ -225,485 +151,319 @@ disallowed policies group. An example of this is shown below:
-lease_duration     1h
-lease_renewable    true
-password           A1a-3pMGjpDXHZ2Qzuf7
-username           v-root-accessdb-5LA65urB4daA8KYy2xku-1542318363
+# Apply the policy to our engineering/frontend namespace
+$ vault policy write -namespace=engineering/frontend nomad-server nomad-server-policy.hcl
 ```
 
-Congratulations! You have configured Vault's connection to your database and can
-now generate credentials with the previously specified privileges. Now we need
-to deploy our application and make sure that it will be able to communicate with
-Vault and obtain the credentials as well.
+We will also configure the previously configured `nomad-cluster` role with each Namespace
 
-### Step 13: Create the `access-tables` Policy for Your Nomad Job to Use
+```shell-session
+# Create the token role with Vaule for engineering
+$ vault write -namespace=engineering /auth/token/roles/nomad-cluster @nomad-cluster-role.json
 
-Recall from [Step 5][step-5] that we specified a policy named `access-tables` in
-our `allowed_policies` section of the token role. We will create this policy now
-and give it the capability to read from the `database/creds/accessdb` endpoint
-(the same endpoint we read from in the previous step to generate credentials for
-our database). We will then specify this policy in our Nomad job which will
-allow it to retrieve credentials for itself to access the database.
+# Create the token role with Vaule for engineering/frontend
+$ vault write -namespace=engineering/frontend /auth/token/roles/nomad-cluster @nomad-cluster-role.json
+```
 
-On the Nomad server (which is also running Vault), create a file named
-`access-tables-policy.hcl` with the following content:
+The [Nomad agent Vault integration][config] congfiguration supports specifying a Vault Namespace, but since
+we will be using multiple and have configured our policies it can be left blank so by default Nomad interacts with Vault in it's root Namespace.
 
 ```hcl
-path "database/creds/accessdb" {
-  capabilities = ["read"]
+vault {
+  enabled               = true
+  ca_path               = "/etc/certs/ca"
+  cert_file             = "/var/certs/vault.crt"
+  key_file              = "/var/certs/vault.key"
+  address               = "https://vault.service.consul:8200"
+  create_from_role      = "nomad-cluster"
+  allow_unauthenticated = false # Disabling allow_unauthenticated is a best practice for securing your cluster
 }
 ```
 
-Create the `access-tables` policy with the following command:
+The same steps can be taken to inject a Vault token from the [Retrieving the Token Role based Token](#retrieving-the-token-role-based-token) steps.
 
-```shell-session
-$ vault policy write access-tables access-tables-policy.hcl
-```
+### Submitting a job with a Vault Namespace
 
-You should see the following output:
+Since allow_unauthenticated is set to false job submitters will need to get a sufficiently privileged token to use when submitting a job.
```

Comment:
```suggestion
Since [`allow_unauthenticated`][allow_unauth] is set to `false` job submitters will need to provide a sufficiently privileged token when submitting a job.

[allow_unauth]: /docs/configuration/vault#allow_unauthenticated
```